src/drawer.splitchannels.js

```diff
@@ -0,0 +1,460 @@
+/**
+ * @since 3.0.0
```

Comment:
this version nr needs to be updated (see changelog for nxt version nr).

Reply:
Ooops, I'll update.